configs/container.config

```diff
@@ -2,7 +2,7 @@ process {
     withLabel:	hisat2	{ container = "nanozoo/hisat2:2.1.0--66dae66" } 
     withLabel:	minimap2	{ container = "nanozoo/minimap2:2.22--b178b67" } 
     withLabel:	python3	{ container = "nanozoo/python_rnaseq:3--23823db" } 
-    withLabel:	deseq2	{ container = "nanozoo/deseq2:1.28.0--0df1612" } 
+    withLabel:	deseq2	{ container = "huxleys/deseq2:1.34.0" } 
```

Comment:
Great, finally an update of DESeq2 in the pipeline : )

I just took the conda yaml and also build the container on nanozoo, see this can just stay harmonized/ using the same base images:

`docker pull nanozoo/deseq2:1.34.0--c670fa0` 

Needs testing though. But I'm first going through the other PRs